src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

```diff
@@ -46,7 +44,7 @@ The browser agent officially supports the following browser versions:
 
 Instrumentation and specific features may be compatible with other browsers or versions.
 
-The user interface is compatible with [New Relic's supported browser versions](/docs/data-analysis/user-interface-functions/view-your-data/supported-browsers-new-relics-ui).
+The user interface is compatible with [supported browser versions](/docs/data-analysis/user-interface-functions/view-your-data/supported-browsers-new-relics-ui).
```

Comment:
```suggestion
```

I think we can remove this. It's redundant with the list above.

Reply:
I agree.